Sources/MapboxMaps/Gestures/GestureHandlers/PinchBehaviors/PinchBehaviorProvider.swift

```diff
@@ -24,18 +26,21 @@ internal final class PinchBehaviorProvider: PinchBehaviorProviderProtocol {
                                     rotateEnabled: Bool,
                                     initialCameraState: CameraState,
                                     initialPinchMidpoint: CGPoint,
-                                    initialPinchAngle: CGFloat) -> PinchBehavior {
+                                    initialPinchAngle: CGFloat,
+                                    focalPoint: CGPoint?) -> PinchBehavior {
         switch (panEnabled, zoomEnabled, rotateEnabled) {
         case (true, true, true):
             return PanZoomRotatePinchBehavior(
                 initialCameraState: initialCameraState,
                 initialPinchMidpoint: initialPinchMidpoint,
                 initialPinchAngle: initialPinchAngle,
+                focalPoint: focalPoint,
                 mapboxMap: mapboxMap)
         case (true, true, false):
             return PanZoomPinchBehavior(
                 initialCameraState: initialCameraState,
                 initialPinchMidpoint: initialPinchMidpoint,
+                focalPoint: focalPoint,
                 mapboxMap: mapboxMap)
         case (true, false, true):
             return PanRotatePinchBehavior(
```

Comment:
I think PanRotatePinchBehavior needs the focalPoint too

Reply:
Or maybe PanZoomRotatePinchBehavior and PanZoomPinchBehavior should *not* have focalPoint

Reply:
It sounds to me that it probably is a question of whether the focal point affects rotation or not. This PR assumes rotation is affected, from this perspective `PanRotatePinchBehavior` missing the `focalPoint` option is an oversight.

Reply:
What's the behavior if you're panning and rotating simultaneously with a fixed focal point?

Reply:
Well... It behaves as if the focal point is not specified.

Reply:
Notes from our call:

Add `didSet` to `GestureOptions.pinchPanEnabled` that sets `focalPoint` to `nil` and logs a warning if `pinchPanEnabled` is `true` and `focalPoint != nil`. Update the documentation to describe this relationship.

Do the same to `GestureOptions.focalPoint`.

Remove `focalPoint` from all `Pan*PinchBehavior` classes

Reply:
Closing this as we decided to disallow having pinch pan enabled and a fixed focal point at the same time.